server/controllers/sharing.coffee

```diff
@@ -12,14 +12,20 @@ mailTemplate = notiftemplate = localization.getEmailTemplate 'sharemail.jade'
 
 clearanceCtl = clearance.controller
     mailTemplate: (options, callback) ->
-        options.type = options.doc.docType.toLowerCase()
-        User.getDisplayName (err, displayName) ->
-            options.displayName = displayName or \
-                                  localization.t 'default user name'
-
+        # Use async to retrieve all wanted informations
+        async.parallel [
```

Comment:
Both values could be retrieve in one request. Display name and email are set at the user object level.